release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -805,6 +805,54 @@ New Fluentd parameters allow you to performance-tune your Fluentd log collector.
 
 These parameters can help you determine the trade-offs between latency and throughput in your cluster logging instance.
 
+[id="ocp-4-6-monitoring"]
+=== Monitoring
+
+[id="ocp-4-6-monitoring-for-user-defined-projects"]
+==== Monitoring for user-defined projects
+
+In {product-title} 4.6, you can enable monitoring for user-defined projects in addition to the default platform monitoring. You can now monitor your own projects in {product-title} without the need for an additional monitoring solution. Using this new feature centralizes monitoring for core platform components and user-defined projects.
+
+With this new feature, you can perform the following tasks:
+
+* Enable and configure monitoring for user-defined projects
+* Create recording and alerting rules that use metrics from your own pods and services
+* Access metrics and information about alerts through a single, multi-tenant interface
+* Cross-correlate the metrics for user-defined projects with platform metrics
+
+For more information, see xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Understanding the monitoring stack].
+
+[id="ocp-4-6-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+The following alerting rule changes have been introduced in {product-title} 4.6:
+
+* The `PrometheusOperatorListErrors` alert has been added. The alert provides notification of errors when running list operations on controllers.
+* The `PrometheusOperatorWatchErrors` alert has been added. The alert provides notification of errors when running watch operations on controllers.
+* The `KubeQuotaExceeded` alert has changed to `KubeQuotaFullyUsed`. Previously, the `KubeQuotaExceeded` alert fired if a ResourceQuota exceeded a 90% threshold. The `KubeQuotaFullyUsed` alert fires if a ResourceQuota is fully used.
+* etcd alerts now support the addition of custom labels for metrics.
+* The `KubeAPILatencyHigh` and `KubeAPIErrorsHigh` alerts have been replaced by the `KubeAPIErrorBudgetBurn` alert. `KubeAPIErrorBudgetBurn` combines API error and latency alerts and only fires when the conditions are severe enough.
+* The readiness and liveness probe metrics exposed by the kubelet are now scraped.
```

Comment:
For what?

Reply:
This provides historical liveness and readiness data for containers, which can be helpful when troubleshooting container issues. I have now added this to the bullet point. Thanks for highlighting this!